docs/overview/policy-lifecycle.mdx

```diff
@@ -4,62 +4,84 @@ sidebar_label: Policy Lifecycle
 
-### Build 
+There are two paths for managing the lifecycle of a policy image:
 
-The set of source files that comprise a policy are built into a **Policy Image** using a set of tools that 
-are modeled after the `docker` toolset.  You can build and tag a policy just like you build and tag a docker image.
+1. Using the `policy` CLI to build, tag and push a policy image to a policy registry.
+2. Using GitHub, a **build action** automatically builds a new version of a policy image when the source repository is **tagged** with a new tag.
 
-:::note
-Currently, the Aserto control plane wires up directly to a git repository that contains policy files, and 
-does not expose the policy image directly. A GitHub **build action** automatically builds a new version of 
-a policy image when the source repository is **tagged** with a new tag:
-:::
+### `policy` CLI
+
+#### Login
 
 ```bash
-git commit -am 'new policy version'
-git tag v0.1.2 && git push --tags
+policy login -s <PATH-TO-REGISTRY> -u <USERNAME> -p <PASSWORD>
 ```
 
-### Publish 
+To connect to the Aserto container registry, use the path `registry.prod.aserto.com`.
+
+#### Build
+
+To build a policy image, use the `policy build` command. Then, tag your built image by using the `policy tag` command.
+
+```bash
+policy build . -t my-org/my-policy:v0.1.2
+policy tag my-org/my-policy:v0.1.2 my-org/my-policy:latest
+```
+
+#### Publish
+
+To publish a built policy image to the policy registry, use the `policy push` command:
+
+```bash
+policy push my-org/my-policy:v0.1.2
+```
+
+#### Pull
+
+Using the `policy` CLI, you can pull a policy image from a Policy Repository just like you pull a docker image.
+
+```bash
+policy pull my-org/my-policy:v0.1.2
+```
 
-You can push a policy image to a **Policy Repository**, which is hosted in a **Policy Registry**, such as `registry.prod.aserto.com`.
+### Github Actions
 
-You can pull a policy image from a Policy Repository just like you pull a docker image.
+Policy repositories generated by Aserto include Github Actions which automatically build and push new policy images to the Aserto Container Registry when a new tag is pushed to the repository.
 
-:::note
-Currently, the Aserto Policy Registry is not exposed using a public API. The GitHub action will build 
-a policy image and push it to the Aserto Policy Registry when the repo is tagged. In the future, the Policy 
-Registry will support a set of public operations very similar to a Docker registry.
-:::
+```bash
+git commit -am 'new policy version'
+git tag v0.1.2 && git push --tags
+```
 
 ---
 
 **GitHub Actions Secret**
 
 Aserto can set up a secret for you when you create a new policy.
 You can also setup this secret yourself, using information from the **Policy Settings** tab in the Aserto Console.
+
 The secret is a JSON object that looks like this:
 
```

Comment:
Probably worth spelling out that this should be the value in the ASERTO_PUSH_KEY GitHub secret.  (also, worth finding out how this may be different now that we have a configuration file that is checked into the .github/workflows directory which contains some of this info, like tenant ID and policy ID).